net-im/qtox/qtox-1.15.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils xdg-utils
+
+MY_P="qTox-${PV}"
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/qTox/qTox"
+SRC_URI="https://github.com/qTox/qTox/archives/v${PV}.tar.gz -> ${MY_P}.tar.gz"
```

Comment:
The 'SRC_URI' here seems incorrect ('archives' versus 'archive'), which leads into an html file to be downloaded.